prov/gni/include/gnix_cntr.h

```diff
@@ -49,6 +51,13 @@ struct gnix_cntr_poll_nic {
 	struct gnix_nic *nic;
 };
 
+enum {
+	GNIX_CNTR_STATE_UNINITIALIZED = 0,
+	GNIX_CNTR_STATE_READY,
+	GNIX_CNTR_STATE_WAITING_ON_REFS,
+	GNIX_CNTR_STATE_DEAD,
+};
```

Comment:
GNIX_CNTR_STATE_DEAD is only used (not set) and GNIX_CNTR_STATE_WAITING_ON_REF isn't used at all.  Are they necessary?